source/Calamari/Integration/Scripting/Bash/BashScriptBootstrapper.cs

```diff
@@ -47,6 +49,18 @@ public static string PrepareConfigurationFile(string workingDirectory, CalamariV
             return configurationFile;
         }
 
+        static string LogEnvironmentInformation()
+        {
+            var environmentInformationStamp = $"Bash Environment Information:{Environment.NewLine}" +
+                $"  {string.Join($"{Environment.NewLine}  ", EnvironmentHelper.SafelyGetEnvironmentInformation())}";
```

Comment:
I think this is not quite what we want... This will tell the bash script to log the context  `Calamari.exe` is currently running in, not necessarily the environment the actual bash script is running in. I could be wrong though!